tools/peptideshaker/peptide_shaker.xml

```diff
@@ -12,8 +12,7 @@
     </requirements>
     <expand macro="stdio" />
 
-    <command><![CDATA[
-    ## When supporting more advanced Galaxy versions:    command use_shared_home="false"
+    <command use_shared_home="false"><![CDATA[
```

Comment:
I think it would be better to increase the `profile` of the tools to something greater than or equal to 18.01. 

See https://github.com/galaxyproject/galaxy/blob/ce23f963e036722865ecd27fda509f173406ab61/lib/galaxy/tool_util/parser/xml.py#L169

I would suggest at least 20.01

Reply:
Yes, you are right @bernt-matthias, thank you. That was something I wanted to do some time ago but I didn't remember to include it...